register.py

```diff
@@ -18,6 +18,7 @@ def main():
     argparse_add_environ(parser, '--etcd-port', type=int, default=2379, help='etcd port')
     argparse_add_environ(parser, '--etcd-prefix', type=str, default='external-dns', help='Prefix for etcd record keys')
     argparse_add_environ(parser, '--loop-every', type=int, default=0, help='Loop every n seconds. Set to 0 to run once')
+    argparse_add_environ(parser, '--owner-id', type=str, default='skydns-register', help='ID of the owner in case more than one skydns-register is running')
```

Comment:
```suggestion
    argparse_add_environ(parser, '--owner-id', type=str, default='skyreg', help='ID of the owner in case more than one skydns-register is running')
```